app/Core/Router.php

```diff
@@ -29,4 +42,74 @@ private function __construct() {
     public static function getInstance() : Router {
         return self::$routerInstance ?? new Router();
     }
+
+    /**
+     *  Get Request on url and action
+     *
+     * @param string url
+     * @param string action
+     */
+    public function get($url, $action){
+        $this->addRoute('Get', $url, $action);
```

Comment:
Could you make the 'Get' a class constant and reference it here, also do the same for the other types.